dandi/metadata.py

```diff
@@ -43,7 +44,63 @@
 
 lgr = get_logger()
 
+# Remove hard-coding when current version fallback is merged.
 
+BIDS_TO_DANDI = {
+    "subject": "subject_id",
+    "session": "session_id",
+}
+
+
+def _rename_bids_keys(bids_metadata, conversion=BIDS_TO_DANDI):
+    """Standardize BIDS metadata field naming to match DANDI."""
+    converted_bids_metadata = {}
+    for key in bids_metadata.keys():
+        if key in conversion:
+            converted_bids_metadata[conversion[key]] = bids_metadata[key]
+        else:
+            converted_bids_metadata[key] = bids_metadata[key]
+    return converted_bids_metadata
+
+
+def _path_in_bids(
+    path, root_marker="dataset_description.json", end_marker="dandiset.yaml"
+):
+    """Determine whether a path is a member of a BIDS dataset.
+
+    Parameters
+    ----------
+    path: str or Path
+    root_marker: str, optional
+        String giving a filename, the existence of which in a directory will mark it as a
+        BIDS dataset root directory.
+    end_marker: str, optional
+        String giving a filename, the existence of which in a directory will end the
+        search.
+
+    Returns
+    -------
+    bool
+    """
+    from .utils import find_files
+
+    path = Path(path)
+    if [i for i in find_files(root_marker, path)]:
+        return True
+    search_path = path
+    while True:
+        search_path = search_path.parent
+        bids_marker_candidate = search_path / root_marker
```

Comment:
align with variable names:
```suggestion
        root_marker_candidate = search_path / root_marker
```
(adjust usage below).